### ISSUE TITLE
Include from GitLab (self-hosted) not possible

### ISSUE THREAD

**Issue by maennchen** [kachick/renovate-config-asdf#206]
If I understand it correctly, the include is not possible because `local` on `github` refers to itself. When using from a self hosted `gitlab`, `local` refers to that one.

I therefore think that the `default` json should import from `github>kachick/renovate-config-asdf//plugins/[PLUGIN].json5#[VERSION]` instead of `local>kachick/renovate-config-asdf//plugins/[PLUGIN].json5`.

**Comment by kachick** [comment]
>I therefore think that the default json should import from github>kachick/renovate-config-asdf//plugins/[PLUGIN].json5#[VERSION] instead of local>kachick/renovate-config-asdf//plugins/[PLUGIN].json5.

Currently `main` uses `local>` and `tagged releases` uses `github>`.  Is it not enough? 

>Therefore, I would guess that the current behavior is buggy since the version tagged refers to the plugins on main instead of on the same tag.

Agreed... 😢 Using 1.4.1 correctly using the plugins, however they are looks loading main branch files.

https://app.renovatebot.com/dashboard#github/kachick/public_dotfiles/849820065

```
DEBUG: http statistics
{
  "urls": {
    "https://api.github.com/graphql (POST,200)": 16,
    "https://api.github.com/repos/denoland/deno/releases (GET,200)": 1,
    "https://api.github.com/repos/dprint/dprint-plugin-json/releases (GET,200)": 1,
    "https://api.github.com/repos/dprint/dprint-plugin-markdown/releases (GET,200)": 1,
    "https://api.github.com/repos/dprint/dprint/releases (GET,200)": 1,
    "https://api.github.com/repos/elm/compiler/releases (GET,200)": 1,
    "https://api.github.com/repos/kachick/public_dotfiles/contents/renovate.json (GET,200)": 1,
    "https://api.github.com/repos/kachick/public_dotfiles/pulls (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/default.json (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/bun.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/clojure.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/crystal.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/deno.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/direnv.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/dprint.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/elixir.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/elm.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/erlang.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/gauche.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/golang.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/haskell.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/helm.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/idris.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/java.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/julia.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/kotlin.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/kustomize.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/lua.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/nim.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/nodejs.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/ocaml.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/perl.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/php.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/python.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/ruby.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/rust.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/scala.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/shellcheck.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/shfmt.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/terraform.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/trivy.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-asdf/contents/plugins/zig.json5 (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-dprint/contents/asdf.json (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-dprint/contents/default.json (GET,200)": 1,
    "https://api.github.com/repos/kachick/renovate-config-dprint/contents/plugins.json (GET,200)": 1,
    "https://api.github.com/repos/koalaman/shellcheck/releases (GET,200)": 1,
    "https://api.github.com/repos/mvdan/sh/releases (GET,200)": 1,
    "https://api.github.com/repos/whitesource/merge-confidence/contents/beta.json (GET,200)": 1,
    "https://api.github.com/repositories/133442384/releases (GET,200)": 1
  },
  "hostStats": {
    "api.github.com": {
      "requestCount": 64,
      "requestAvgMs": 337,
      "queueAvgMs": 0
    }
  },
  "totalRequests": 64
}
```

One of my idea is gathering the plugins to tagged default.json with scripts. 🤔

**Comment by maennchen** [comment]
@kachick Oh, I did not realize that the tags have a different `extends` from `github`.

I tried it using `1.4.0` yesterday. That one was still using `local`. I'll try it again using `1.6.0` and see if this issue is still present.

**Comment by maennchen** [comment]
@kachick Sorry for the noise. YOu fixed it with `1.4.1`.
